### DISCUSSION TITLE
摄像头增加camera实体

### DISCUSSION THREAD

**Discussion by silas3082** [XiaoMi/ha_xiaomi_home#128] [Ideas]
现在的摄像头里边只有传感器、按钮、开关的实体，没有摄像头的实体。
![设置 – Home Assistant 和另外 1 个页面 - 用户1 - Microsoft​ Edge 2024_12_18 9_26_31](https://github.com/user-attachments/assets/56b37f28-330e-46ba-989a-cb5f8a5e87e3)

设置里边有：”P2P视频流 开始实时视频流“按钮，是不是意味着摄像头的P2P视频流 也能获取到，并且能在ha中查看摄像头的实时画面？

**Comment by ispacc** [comment]
说是因为摄像头不支持串流不好实现

**Reply by silas3082** [reply]
希望后期小米的工程师们能给接入，这样玩法就更多了

**Comment by SusanPhevos** [comment]
See #78
